src/main/scala/ing/wbaa/druid/client/DruidAdvancedHttpClient.scala

```diff
@@ -77,27 +78,29 @@ class DruidAdvancedHttpClient private (
 
     val request = HttpRequest(HttpMethods.GET, uri = druidConfig.healthEndpoint)
 
-    val checksF = Future.sequence {
+    val checksF: Future[Iterable[(QueryHost, Boolean)]] = Future.sequence {
       brokerConnections.map {
         case (queryHost, flow) =>
-          val responsePromise = Promise[HttpResponse]()
-
-          Source
-            .single(request -> responsePromise)
-            .via(flow)
-            .runWith(Sink.head)
-            .flatMap {
-              case (Success(response), _) =>
-                Future {
-                  response.discardEntityBytes()
-                  queryHost -> (response.status == StatusCodes.OK)
-                }
-              case (Failure(ex), _) => Future.failed(ex)
+          healthLogger.debug(
```

Comment:
Since this PR also makes some improvements to the logging. Perhaps it would be better to switch to `.debug("... {} ... {} ... {}", param1, param2)` syntax and  avoid any string concatenation if the corresponding logging level is suppressed.

Reply:
I tried, and I found that calling Java varargs from scala code introduces a lot of noise. I will convert the code from `org.slf4j` logging to `com.typesafe.scala-logging` and bypass the entire problem. Since that goes quite far out of scope for this PR, I will create a separate PR for the purpose.